### PR TITLE
bedtools: add page

### PR DESCRIPTION
**Pull request by mashehu**
First try for a bedtools page, see #1168

### CODE REVIEW

**Comment by agnivade** [comment]
Fixes #1168

**Comment by mashehu** [comment]
Just wondering if anything is still missing from my side to get the pull request finished.

File: pages/common/bedtools.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`bedtools intersect -a {{path/to/file_1}} -b {{path/to/file_2}} -sorted > {{path/to/output_file}}`
+
+- Group file {{path/to/file}} based on the first three and the fifth column and summarize the sixth column by summing it up:
+
+`bedtools groupby -i {{path/to/file}} -c 1-3,5 -g 6 -o sum`
+
+- Convert bam-file to bed-formated file:
+
+`bedtools bamtobed -i {{path/to/file.bam}} > {{path/to/file.bed}}`
```
Comment (agnivade) [inline pages/common/bedtools.md]: .bam and .bed should be outside the tokens.

File: pages/common/bedtools.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Find for all features in {{file_1.bed}} the closes one in {{file_2.bed}} and write their distance in an extra column (input files must be sorted):
```
Comment (agnivade) [inline pages/common/bedtools.md]: closes => typo

File: pages/common/bedtools.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# bedtools
+
+> A swiss-army knife of tools for genomic-analysis tasks.
+> Used to intersect, group, convert and count data in  BAM, BED, GFF/GTF, VCF format.
+
+- Intersect two files strand-specific and save the result to {{path/to/output_file}}:
```
Comment (agnivade) [inline pages/common/bedtools.md]: "files strand-specific" - the grammar doesn't feel right with this one. Shouldn't it be strand-specific files ?

**Comment by agnivade** [comment]
LGTM

**Comment by waldyrious** [comment]
I normally would've added some nitpicking comments (minor stuff), but I'm not sure why this went under my radar. Since it's been waiting for quite a while, and there are no major issues, I'll forgo that, give my thumbs-up and merge. Thanks @mashehu!

**Pull request merged**